packages/react/src/screens/core/UserDetails.tsx

```diff
@@ -0,0 +1,817 @@
+import { LoginIdentifierConfigType, LoginIdentifierType } from '@corbado/shared-ui';
```

Comment:
We should break this up into components :)
Let's get help from @Dopeamin for that